src/retuner.cc

```diff
@@ -34,7 +34,8 @@ Retuner::Retuner (int fsamp) :
     _corrfilt (1.0f),
     _corrgain (1.0f),
     _corroffs (0.0f),
-    _notemask (0xFFF)
+    _notemask (0xFFF),
+    _notescale {0.0f, 1.0f, 2.0f, 3.0f, 4.0f, 5.0f, 6.0f, 7.0f, 8.0f, 9.0f, 10.0f, 11.0f}
```

Comment:
This extended initializer list is only available with C++11 or later.
The plugin works on much older systems.  Check with `make CXXFLAGS="-std=c++98"`